arviz/plots/pointwiseelpdplot.py

```diff
@@ -0,0 +1,242 @@
+"""Plot pointwise elpd estimations of inference data."""
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+from matplotlib.ticker import NullFormatter
+from matplotlib.lines import Line2D
+
+from ..data import convert_to_inference_data
+from .plot_utils import _scale_fig_size
+from ..stats import waic, loo
+
+
+def plot_pointwise_elpd(
+    idata_dict,
+    ic="waic",
+    color=None,
+    xlabels=False,
+    figsize=None,
+    textsize=None,
+    coords=None,
+    legend=False,
+    ax=None,
+    scale="deviance",
+    plot_kwargs=None,
+):
+    """
+    Plot a scatter or hexbin matrix of the sampled parameters.
+
+    Parameters
+    ----------
+    idata_dict : mapping, str -> InferenceData
+        A dictionary mapping the model name to the object containing its inference data.
+        Refer to az.convert_to_inference_data for details on possible dict items
+    ic : str, optional
+        Information Criterion (WAIC or LOO) used to compare models. Default WAIC.
```

Comment:
Is there a reason why waic is the default?

Reply:
I followed the default in `az.compare`, maybe it can be included as a package option too?

Reply:
Sounds ok, let's follow `az.compare`